charmander/app.py

```diff
@@ -7,10 +7,10 @@
 from microcosm.loaders.compose import load_config_and_secrets
 from microcosm_secretsmanager.loaders.conventions import load_from_secretsmanager
 
+import charmander.factories.order_event_factory  # noqa
 import charmander.postgres  # noqa
-import charmander.routes.pizza.controller  # noqa
-import charmander.routes.pizza.crud  # noqa
-import charmander.stores.pizza_store  # noqa
+import charmander.routes  # noqa
```

Comment:
For the graph initialization on L32 to work, we need all of the factories referenced by the graph to be in scope.

Even a service with a few resources can end up with many routes and stores so we usually end up funneling all imports into the `__init__.py` for the relevant package and importing them all at once.

See further below for notes where this is being done.